$NetBSD: patch-aa,v 1.3 2000/06/30 19:55:06 dmcmahill Exp $

use a default timescale of 1ns instead of 1ps.  This will make the VCD output
compatible with Dinotrace.

--- vpi/sys_vcd.c.orig	Fri Jun  2 22:22:15 2000
+++ vpi/sys_vcd.c	Fri Jun 30 15:51:40 2000
@@ -252,5 +252,5 @@
 	    fprintf(dump_file, "$end\n");
 	    fprintf(dump_file, "$timescale\n");
-	    fprintf(dump_file, "\t1ps\n");
+	    fprintf(dump_file, "\t1ns\n");
 	    fprintf(dump_file, "$end\n");
       }
